tests/MiddlewareTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace Spatie\Permission\Test;
+
+use Auth;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Spatie\Permission\Middleware\RoleMiddleware;
+use Spatie\Permission\Middleware\PermissionMiddleware;
+
+class MiddlewareTest extends TestCase
+{
+    /**
+     * @test
+     * @expectedException Symfony\Component\HttpKernel\Exception\HttpException
```

Comment:
I'm not a fan of using the `expectedException` annotation. Could you use the equivalent method instead?